client/src/store/historyStore/historyItemsFiltering.test.js

```diff
@@ -0,0 +1,74 @@
+import { getFilters, getQueryDict, testFilters } from "./historyItemsFiltering";
+
+const filterTexts = [
+    "name='name of item' hid>10 hid<100 create-time>'2021-01-01' update-time<\"2022-01-01\" state=success extension=ext tag=first",
+    "name='name of item' hid>10 hid<100 create_time>\"2021-01-01\" update_time-lt='2022-01-01' state=success extension=ext tag=first",
```

Comment:
I guess you wanted to try more alternative syntax cases here?
```suggestion
    "name='name of item' hid_gt=10 hid-lt=100 create_time-gt=\"2021-01-01\" update_time-lt='2022-01-01' state=success extension=ext tag=first",
```